### PR TITLE
Add backwards compatibility for menu-state-active & menu-centered

### PR DESCRIPTION
**Pull request by IamManchanda**
Keep `active` class but with option to disable the same!

cc @kball for merge/review!

### CODE REVIEW

File: scss/components/_menu.scss [hunk 1]
```diff
@@ -437,6 +458,19 @@ $menu-item-background-hover: $light-gray !default;
     }
   }
 
+  @if $menu-centered-back-compat {
+    .menu-centered {
+      > .menu {
+        @if $global-flexbox {
+          display: flex;
```
Comment (kball) [inline scss/components/_menu.scss]: Why do we need to add display flex? In global flexbox case won't we already be display flex?

**Comment by IamManchanda** [comment]
For test case 

http://localhost:3000/menu/active-menu-flex.html
http://localhost:3000/menu/active-menu-float.html

http://localhost:3000/menu/menu-centered-flex.html
http://localhost:3000/menu/menu-centered-float.html

**Comment by IamManchanda** [comment]
You are right @kball ... removed! 
KIndly check now ... above are the links for test case!

**Comment by kball** [comment]
👍

**Pull request merged**